src/Utilities/functions.jl

```diff
@@ -146,8 +155,11 @@ function map_indices(
     index_map::F,
     t::MOI.ScalarQuadraticTerm,
 ) where {F<:Function}
-    inds = index_map.((t.variable_1, t.variable_2))
-    return MOI.ScalarQuadraticTerm(t.coefficient, inds...)
```

Comment:
Where possible, we should avoid splatting. The change below is more readable also.